packages/@n8n/chat/src/css/markdown.scss

```diff
@@ -641,4 +640,15 @@ body {
 	body > a:first-child:focus {
 		top: 1rem;
 	}
+
+	// Lists
+	ul,
+	ol {
+		padding-left: 1.5rem;
```

Comment:
Nit: Can we use spacing tokens where possible?

Reply:
We don't have the standard spacing tokens from n8n here :/ Because chat-sdk is a standalone package.